site/docs/guidelines/checkboxs-radiobuttons-toggles.mdx

```diff
@@ -0,0 +1,131 @@
+---
+name: Checkboxes, radio buttons or toggles?
+route: /guidelines/checkboxes-radiobuttons-toggles
+menu: Guidelines
+---
+
+import LargeParagraph from "../../src/components/LargeParagraph";
+import { Notification, Checkbox, RadioButton, ToggleButton, SelectionGroup } from "hds-react";
+import Link from "../../src/components/Link";
+import { Playground } from 'docz';
+
+# Checkboxes, radio buttons or toggles?
+
+<LargeParagraph>
+    Helsinki Design System has checkboxes, radio buttons and toggles that might seem similar to each other, but it is important to consider the use cases to decide which one to use.
+</LargeParagraph>
+
+<Notification label="TLDR (Too long, didn't read)" className="siteNotification">
+<b>Use checkboxes</b> to select multiple parallel items. Use checkbox to single selections to confirm on/off selection.
+<br /><br />
+<b>Use radio buttons</b> to mutually exclusive options.
+<br /><br />
+<b>Use toggle</b> to selections affecting system settings or changing status.
+</Notification>
+
+
+## When to use radio buttons?
+<Link href="../components/radio-button" >HDS radio button component</Link>
+<br /><br />
+An application can use radio buttons when there is a list of two or more mutually exclusive choices and the user needs to select one of them. After clicking the non-selected radio button, whatever was previously selected is deselected. In traditional application design, the first radio button in the list was always selected by default. But modern website design challenged this practice, making it fairly common to have none selected by default. Websites today, forms, and applications inconsistently select one or leave all radio buttons blank by default. If none of the buttons is selected by default, users have no way to revert to this original state after they’ve made a selection. The lack of a standard can be confusing to users.
```

Comment:
My language tool did not like the sentence "But modern website design challenged.." but I'm not sure if there is anything wrong 🤔 
Would it be better as "But the modern web design challenged.."